scripts/gen_vpc_ip_limits.go

```diff
@@ -128,65 +134,76 @@ func describeRegions() []string {
 func describeInstanceTypes(region string, eniLimitMap map[string]vpc.InstanceTypeLimits) {
 	log.Infof("Describing instance types in region=%s", region)
 
-	// Get session
-	sess := session.Must(session.NewSessionWithOptions(session.Options{
-		SharedConfigState: session.SharedConfigEnable,
-		Config:            *aws.NewConfig().WithRegion(region),
-	}))
-	_, err := sess.Config.Credentials.Get()
+	// Create context
+	ctx := context.Background()
+
+	// Load AWS configuration with specific region
+	cfg, err := config.LoadDefaultConfig(ctx,
```

Comment:
JFYI you'll probably get more mileage in general out of having your config loading logic be reused and passed around (or just pass around the config itself).